cypress/integration/e2e/consent.spec.js

```diff
@@ -0,0 +1,93 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+
+const firstPage =
+    'https://www.theguardian.com/environment/2020/oct/13/maverick-rewilders-endangered-species-extinction-conservation-uk-wildlife';
+
+const secondPage =
+    'https://www.theguardian.com/environment/2020/nov/19/blue-whale-sightings-off-south-georgia-raise-hopes-of-recovery';
+
+describe('Consent tests', function () {
+    before(getPolyfill);
+
+    const cmpIframe = () => {
+        return cy
+            .get('iframe[id*="sp_message_iframe"]')
+            .its('0.contentDocument.body')
+            .should('not.be.empty')
+            .then(cy.wrap);
+    };
+
+    const privacySettingsIframe = () => {
+        return cy
+            .get('[src*="https://cdn.privacy-mgmt.com/privacy-manager"]')
+            .its('0.contentDocument.body')
+            .should('not.be.empty')
+            .then(cy.wrap);
+    };
+
+    const waitForAnalyticsToInit = () => {
```

Comment:
Okay, yes, this makes sense. At some point we just need to pick an amount of time to wait before declaring that something hasn't happened. This 300ms is much nicer than the seconds (odd) we had before